src/redux/tender/reducers.tsx

```diff
@@ -1,62 +1,60 @@
-import {FilterRequest, Page, Tender} from "../../model";
-import {createSlice, PayloadAction} from "@reduxjs/toolkit";
-import {getTenders} from "./actions";
-import {TenderDTO} from "../../api/paperChannel";
-
-
-
-
-const initialState = {
-  loading: false,
-  allData: {} as Page<Tender>,
-  selected: {} as TenderDTO,
-  pagination: {
-    page:1,
-    tot:10,
-    force: false
-  } as FilterRequest,
-}
-
-const tenderSlice = createSlice({
-  name: 'tenderSlice',
-  initialState,
-  reducers : {
-    resetState: () => initialState,
-    resetAllTenderState: (state) => {
-      state.pagination = {
-        ...initialState.pagination,
-        force: !!(state.pagination.force) ? !state.pagination.force : true

```

Comment:
I guess this equals to just `!state.pagination.force`. Please check *only* if testing is feasible.